### PR TITLE
feat: Add StartTimestamp and EndTimestamp for Cron Scheduler

### LINKED ISSUE #4863: Add startTimestamp and endTimestamp to support-cron-scheduler
# 🚀 Feature Request

### Relevant Package [**REQUIRED**]

 This feature request is for support-cron-scheduler

### Description [**REQUIRED**]
Add startTimestamp and endTimestamp to ScheduleJob could help users manage the jobs easier.

### Describe the solution you'd like
Add two int64 field startTimestamp and endTimestamp to ScheduleJob, and they are in Unix millionseconds format.
Leverage the golang time package to control the job start and end.
https://pkg.go.dev/time#AfterFunc

### PR DESCRIPTION
**Pull request by jackchenjc**
closes #4863

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [x] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

File: internal/support/cronscheduler/application/schedulejob.go [hunk 1]
```diff
@@ -39,10 +40,7 @@ func AddScheduleJob(ctx context.Context, job models.ScheduleJob, dic *di.Contain
 	}
 
 	if job.AdminState == models.Unlocked {
```
Comment (cloudxxx8) [inline internal/support/cronscheduler/application/schedulejob.go:42]: move the admin state check into the `handleScheduleJob(ctx, job, dic)`

File: internal/support/cronscheduler/application/schedulejob.go [hunk 2]
```diff
@@ -230,3 +225,45 @@ func LoadScheduleJobsToSchedulerManager(ctx context.Context, dic *di.Container)
 
 	return nil
 }
+
+// handleScheduleJob handles the schedule job based on the startTimestamp and endTimestamp
+func handleScheduleJob(ctx context.Context, job models.ScheduleJob, dic *di.Container) {
```
Comment (cloudxxx8) [inline internal/support/cronscheduler/application/schedulejob.go:230]: rename to `arrangeScheduleJob`

File: internal/support/cronscheduler/application/schedulejob.go
Comment (cloudxxx8) [inline internal/support/cronscheduler/application/schedulejob.go:1]: you missed the logic change in `LoadScheduleJobsToSchedulerManager`

**Review by cloudxxx8 — CHANGES_REQUESTED** [review]

File: internal/support/cronscheduler/application/schedulejob.go
Comment (jackchenjc) [reply internal/support/cronscheduler/application/schedulejob.go:1]: Applied the change to `LoadScheduleJobsToSchedulerManager` and tested.

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**